wfe/web-front-end.go

```diff
@@ -188,6 +188,10 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h wfe
 	})
 }
 
+func marshalIndent(v interface{}) ([]byte, error) {
+	return json.MarshalIndent(v, "", "  ")
```

Comment:
I'd like to stick with spaces. It avoids excessive indentation on terminals where tab is 8 spaces.